content/blog/2020-05-dplyr-1-0-0-last-minute-additions/index.Rmarkdown

```diff
@@ -0,0 +1,147 @@
+---
+title: 'dplyr 1.0.0: last minute additions'
+author: Hadley Wickham
+date: '2020-05-06'
+slug: dplyr-1-0-0-last-minute-additions
+categories:
+  - package
+tags:
+  - dplyr
+description: >
+    Learn about two last minute additions to dplyr 1bl.0.0: a chattier 
+    `summarise()` with more options for controlling grouping of output,
+    and new row manipulation functions inspired by SQL.
+photo:
+  url: https://unsplash.com/photos/FfbVFLAVscw
+  author: Malvestida Magazine
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(collapse = TRUE, comment = "#>")
+```
+
+This post is the latest in a series of post leading up the the dplyr 1.0.0 release on May 15. So far, the series has covered:
+
+-   [Major lifecycle changes](https://www.tidyverse.org/blog/2020/03/dplyr-1-0-0-is-coming-soon/).
+-   [New `summarise()` features](https://www.tidyverse.org/blog/2020/03/dplyr-1-0-0-summarise/).
+-   [`select()`, `rename()`, and (new) `relocate()`](https://www.tidyverse.org/blog/2020/03/dplyr-1-0-0-select-rename-relocate/).
+-   [Working `across()` columns](https://www.tidyverse.org/blog/2020/04/dplyr-1-0-0-colwise/).
+-   [Working within rows](https://www.tidyverse.org/blog/2020/04/dplyr-1-0-0-rowwise/).
+-   [The role of the vctrs package](https://www.tidyverse.org/blog/2020/04/dplyr-1-0-0-and-vctrs/)
+-   [Notes for package developers](https://www.tidyverse.org/blog/2020/04/dplyr-1-0-0-package-dev/)
+
+Today I wanted to talk about two cool new features that we've added since I started blogging about dplyr 1.0.0: `summarise()` now gives you greater control over how the results and grouped, and we've added a new set of functions for modifying rows.
```

Comment:
```suggestion
Today I wanted to talk about two cool new features that we've added since I started blogging about dplyr 1.0.0: `summarise()` now gives you greater control over how the results are grouped, and we've added a new set of functions for modifying rows.
```

Reply:
There's weird list (dis)agreement above. You could use the "we've added" in the beginning, e.g., "we've added greater control over how the results are grouped to `summarise()`, and a new set of functions for modifying rows", but I don't love that. Alternately, just go with "..., and a new set of functions for modifying rows"?

Reply:
Oh yeah, that's messed up. I went with:

Today I wanted to talk about two cool new features that we've added since I started blogging about dplyr 1.0.0: `summarise()` now gives you greater control over how the results are grouped, and a new set of functions make it easier to modify rows.